src/mixins/createRenderMixin.ts

```diff
@@ -149,6 +151,7 @@ function generateID(cachedRender: RenderMixin<RenderMixinState>): string {
 const widgetClassesMap = new WeakMap<RenderMixin<RenderMixinState>, string[]>();
 
 const createRenderMixin = createStateful
+	.mixin(createI18nMixin)
```

Comment:
I know we sort of talked about this, but this then assumes that EVERY widget will be an i18n widget.  Do we want that hard dependency, since it does feel like `RenderMixin` doesn't have any real dependency on I18N from what I can see.